homework/index.js

```diff
@@ -0,0 +1,89 @@
+function loadFromLocalStorage() {
+  const reviewListString = localStorage.getItem("reviews");
+  const reviewList = JSON.parse(reviewListString);
+  if (!reviewList) {
+    return [];
+  }
+  return reviewList;
+}
+
+function saveToLocalStorage(reviewList) {
+  const reviewListString = JSON.stringify(reviewList);
+  localStorage.setItem("reviews", reviewListString);
+}
+
+reviewList = [];
+reviewList = loadFromLocalStorage();
+
+function createReview(review) {
+  const reviewCard = document.createElement("div");
+  reviewCard.className = "reviewCard";
+
+  const reviewMessage = document.createElement("pre");
+  reviewMessage.innerHTML = review.message;
+  reviewCard.appendChild(reviewMessage);
+
+  const reviewRating = document.createElement("p");
+  reviewRating.innerHTML = review.rating + "/5";
+  reviewRating.className = "reviewRating";
+  reviewCard.appendChild(reviewRating);
+
+  const deleteButton = document.createElement("button");
+  deleteButton.innerHTML = "Remove";
+  deleteButton.className = "deleteButton";
+  deleteButton.onclick = () => {
+    deleteReview(review);
+  };
```

Comment:
Za one koji žele znati više 😄 
Proučiti event delegation, pa više ne bi morali na svaki novi review kačit event listener, nego imati samo jedan event listener koji bi slušao klikanje na "Remove" button, kupio neki njegov id i tako ga micao iz DOM-a